playground/optimize-deps/dep-with-builtin-module-cjs/index.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs')
+const path = require('path')
+
+// NOTE: require destructure would error immediately to to how esbuild compiles
+// it. There's no way around as it's direct property access, which triggers
+// the Proxy get trap.
```

Comment:
```suggestion
// NOTE: require destructure would error immediately because of how esbuild
// compiles it. There's no way around it as it's direct property access, which
// triggers the Proxy get trap.
```

Reply:
> There's no way around it as it's direct property access, which
// triggers the Proxy get trap

Proxy get handler can return another Proxy instance, that throws on any kind of access (get, call etc.) 🤔
(Anything besides get handler can throw directly thought, as there may be `module.exports = () => ...` stuff)

Reply:
Thas way modules using named imports from `browser-external` would throw on usage rather than on import.